MonoGame.Framework/Input/Joystick.Default.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.Xna.Framework.Input
 {
     static partial class Joystick
     {
+        private static bool PlatformIsSupported = false;
```

Comment:
@cra0zy - How about making these `const` or `readonly` so that it is clear this shouldn't be changed at runtime?